frontend/src/components/BigPopup/index.tsx

```diff
@@ -31,7 +31,7 @@ export default function BigPopup(props: PopupProps) {
             <ArrowLeftIcon />
           </StyledArrowLeftButton>
           <Box sx={{ width: '35rem' }}>
-            <PointPopup />
+            <PointPopup toggleClose={toggleClose} />
```

Comment:
i can`t find button 'delete' - how you design removing logic of personal location?

Reply:
You can not delete nocation which is  not yours
I did not design removing logic of personal location. it was not in my task. so i didn`t knew i suppose to design it